Tests/test_imagefont.py

```diff
@@ -840,6 +840,102 @@ def test_anchor_invalid(self):
                 ValueError, lambda: d.multiline_text((0, 0), "foo\nbar", anchor=anchor)
             )
 
+    def test_standard_embedded_color(self):
+        txt = "Hello World!"
+        ttf = ImageFont.truetype(FONT_PATH, 40, layout_engine=self.LAYOUT_ENGINE)
+        ttf.getsize(txt)
+
+        img = Image.new("RGB", (300, 64), "white")
+        d = ImageDraw.Draw(img)
+        d.text((10, 10), txt, font=ttf, fill="#fa6", embedded_color=True)
+
+        with Image.open("Tests/images/standard_embedded.png") as expected:
+            assert_image_similar(img, expected, max(self.metrics["multiline"], 3))
+
+    @pytest.mark.xfail(is_pypy(), reason="failing on PyPy with Raqm")
+    def test_cbdt(self):
+        if parse_version(features.version_module("freetype2")) < parse_version("2.5.0"):
+            pytest.skip("Freetype 2.5.0 or newer required")
+        try:
+            font = ImageFont.truetype(
+                "Tests/fonts/NotoColorEmoji.ttf",
+                size=109,
+                layout_engine=self.LAYOUT_ENGINE,
+            )
+
+            im = Image.new("RGB", (150, 150), "white")
+            d = ImageDraw.Draw(im)
+
+            d.text((10, 10), "\U0001f469", embedded_color=True, font=font)
+
+            with Image.open("Tests/images/cbdt_notocoloremoji.png") as expected:
+                assert_image_similar(im, expected, self.metrics["multiline"])
+        except IOError as ex:
+            assert str(ex) in ("unimplemented feature", "unknown file format")
+            pytest.skip("freetype compiled without libpng or unsupported")
+
+    @pytest.mark.xfail(is_pypy(), reason="failing on PyPy with Raqm")
+    def test_cbdt_mask(self):
+        if parse_version(features.version_module("freetype2")) < parse_version("2.5.0"):
+            pytest.skip("Freetype 2.5.0 or newer required")
+        try:
+            font = ImageFont.truetype(
+                "Tests/fonts/NotoColorEmoji.ttf",
+                size=109,
+                layout_engine=self.LAYOUT_ENGINE,
+            )
+
+            im = Image.new("RGB", (150, 150), "white")
+            d = ImageDraw.Draw(im)
+
+            d.text((10, 10), "\U0001f469", "black", font=font)
+
+            with Image.open("Tests/images/cbdt_notocoloremoji_mask.png") as expected:
+                assert_image_similar(im, expected, self.metrics["multiline"])
+        except IOError as ex:
+            assert str(ex) in ("unimplemented feature", "unknown file format")
+            pytest.skip("freetype compiled without libpng or unsupported")
+
+    def test_colr(self):
+        if parse_version(features.version_module("freetype2")) < parse_version(
+            "2.10.0"
+        ):
+            pytest.skip("Freetype 2.10.0 or newer required")
```

Comment:
There's quite a few Freetype version check and skips, let's put them into a helper, for example:

```python
def skip_unless_freetype(required_version, message):
```

Reply:
I went with the following to let the check be reusable for other features:
```python
def skip_unless_feature_version(feature, version_required, reason=None):
```